pandas/tests/arrays/interval/test_interval.py

```diff
@@ -65,3 +65,23 @@ def test_repr_matches():
     a = repr(idx)
     b = repr(idx.values)
     assert a.replace("Index", "Array") == b
+
+
+def test_point_interval():
```

Comment:
make the error conditions as a separate parmeterized test

Reply:
done